src/rules/graphql/no-more-than-100-fields.ts

```diff
@@ -0,0 +1,425 @@
+                                                    value
+                                                }
+                                                Field89 {
+                                                    value
+                                                }
+                                                Field90 {
+                                                    value
+                                                }
+                                                Field91 {
+                                                    value
+                                                }
+                                                Field92 {
+                                                    value
+                                                }
+                                                Field93 {
+                                                    value
+                                                }
+                                                Field94 {
+                                                    value
+                                                }
+                                                Field95 {
+                                                    value
+                                                }
+                                                Field96 {
+                                                    value
+                                                }
+                                                Field97 {
+                                                    value
+                                                }
+                                                Field98 {
+                                                    value
+                                                }
+                                                Field99 {
+                                                    value
+                                                }
+                                                Field100 {
+                                                    value
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    `
+                }
+            ]
+        },
+        messages: {
+            [NO_MORE_THAN_100_FIELDS_RULE_ID]:
+                'Offline GraphQL: The root "{{entityName}}" entity query contains {{numberOfFields}} fields and requests {{numberOfRecords}} records. If a root entity query has more than 100 fields and requests over 200 records, the query will be capped to 200 records'
+        },
+        schema: []
+    },
+
+    create(context: GraphQLESLintRuleContext) {
+        return {
+            Field(node) {
+                // report lint issue if the graphql is like '... uiapi { aggregate { ...'
```

Comment:
irrelevant comment, could be removed or updated

Reply:
removed